tests/fixtures/insert_mock_data.sql

```diff
@@ -31,6 +31,76 @@ VALUES
         NULL
     );
 
+-- disable integrity checks for weight table
+ALTER TABLE weight DISABLE TRIGGER ALL;
+
+-- create dummy weights for users
+INSERT INTO
+    weight (user_id, weight, measured_at)
+VALUES (
+    'user_id_1',
+    123.0,
+    '2020-03-17 12:00:00'
```

Comment:
The mocks use "2020-03-17 12:00:00" but the tests check for "2020-01-20T12:00:00+00:00" and fail

Reply:
@boiledbuns Can you show a screenshot of the failed test? My tests are not failing. Line 41 is inserting a record with the user id `user_id_1`, but the Postman tests use the user id `test_id`.